plugins/versionpress/versionpress.php

```diff
@@ -10,972 +10,38 @@
-    }, 10, 2);
-
-    add_filter('upgrader_pre_download', function ($reply, $_, $upgrader) use ($committer) {
-        if (!isset($upgrader->skin->language_update)) {
-            return $reply;
-        }
-
-        $languages = get_available_languages();
-
-        $postInstallHook = function ($_, $hook_extra) use ($committer, $languages, &$postInstallHook) {
-            require_once(ABSPATH . 'wp-admin/includes/translation-install.php');
-
-            if (!isset($hook_extra['language_update_type'])) {
-                return;
-            }
-
-            $translations = wp_get_available_translations();
-
-            $type = $hook_extra['language_update_type'];
-            $languageCode = $hook_extra['language_update']->language;
-            $languageName = isset($translations[$languageCode])
-                ? $translations[$languageCode]['native_name']
-                : 'English (United States)';
-
-            $name = $type === "core" ? null : $hook_extra['language_update']->slug;
-
-            $action = in_array($languageCode, $languages) ? "update" : "install";
-            $committer->forceChangeInfo(new TranslationChangeInfo($action, $languageCode, $languageName, $type, $name));
-            remove_filter('upgrader_post_install', $postInstallHook);
-        };
-
-        add_filter('upgrader_post_install', $postInstallHook, 10, 2);
-        return false;
-    }, 10, 3);
-
-    add_action('switch_theme', function () use ($committer) {
-        if (defined('WP_CLI') && WP_CLI) {
-            wp_remote_get(admin_url()); //
-        } else {
-            $committer->disableCommit(); // the change will be committed on next load
-        }
-    });
-
-    add_action('after_switch_theme', function () use ($committer) {
-        $theme = wp_get_theme();
-        $stylesheet = $theme->get_stylesheet();
-        $themeName = $theme->get('Name');
-
-        $committer->forceChangeInfo(new ThemeChangeInfo($stylesheet, 'switch', $themeName));
-    });
-
-
-    add_action('wp_ajax_save-widget', function () use ($committer) {
-        if (defined('DOING_AJAX') && DOING_AJAX && isset($_POST['delete_widget']) && $_POST['delete_widget']) {
-            $committer->postponeCommit('widgets');
-        }
-    }, 0); // zero because the default WP action with priority 1 calls wp_die()
-
-    function _vp_get_language_name_by_code($code)
+    function vp_trigger_old_php_error()
     {
-        require_once(ABSPATH . 'wp-admin/includes/translation-install.php');
-
-        $translations = wp_get_available_translations();
-        return isset($translations[$code])
-            ? $translations[$code]['native_name']
-            : 'English (United States)';
+        wp_die('<h1>VersionPress could not be activated</h1>
+            <p>
+                You are using an unsupported version of PHP. We recommend using one of the
+                <a href="http://php.net/supported-versions.php">actively supported</a>.
+            </p>');
```

Comment:
I think that the message should inform user that the PHP version on his server is unsupported by VersionPress itself. Version he is using may be fine for his other business.